pages/config.md

```diff
@@ -1,16 +1,16 @@
 ---
 permalink: /updating-the-config-file/
-title: Updating the config file
+title: Update the config file
 ---
-Work your way through these steps to configure the template for your new guide
-by updating the `_config.yml` file:
+Work your way through these steps to update 
+the `_config.yml` file — this configures your guide's template:
```

Comment:
Just realized that my original text was misleading, as the idea is that the template becomes the new guide, not another template; I'll push an update.